Packs/CyberArk/Integrations/CyberArk/CyberArk.yml

```diff
@@ -0,0 +1,880 @@
+      isArray: false
+      name: platformID
+      required: true
+      secret: false
+    - default: false
+      description: The Safe where the account will be created.
+      isArray: false
+      name: safe_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: password
+      description: The type of password.
+      isArray: false
+      name: secret_type
+      predefined:
+      - password
+      - ' key'
+      required: false
+      secret: false
+    - default: false
+      description: Account user's name.
+      isArray: false
+      name: username
+      required: true
+      secret: false
+    - default: false
+      description: The password value.
+      isArray: false
+      name: password
+      required: true
+      secret: true
+    - default: false
+      description: |-
+        Object containing key-value pairs to associate with the account, as defined by the account platform.
+        e.g.- {"Location": "IT", "OwnerName": "MSSPAdmin"}
+      isArray: false
+      name: properties
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'true'
+      description: Whether the account secret is automatically managed by the CPM.
+      isArray: false
+      name: automatic_management_enabled
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Reason for disabling automatic secret management.
+      isArray: false
+      name: manual_management_reason
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        List of remote machines, separated by semicolons.
+        e.g.- server1.cyberark.com;server2.cyberark.com
+      isArray: false
+      name: remote_machines
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether or not to restrict access only to specified remote machines
+      isArray: false
+      name: access_restricted_to_temote_machines
+      predefined:
+      - 'true'
+      - ' false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Add a new privileged account or SSH key to the Vault.
+    execution: false
+    name: cyberark-pas-add-account
+  - arguments:
+    - default: false
+      description: The unique ID of the account to delete.
+      isArray: false
+      name: accountID
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete a specific account in the Vault
+    execution: false
+    name: cyberark-pas-delete-account
+  - arguments:
+    - default: false
+      description: The unique ID of the account to delete.
+      isArray: false
+      name: accountID
+      required: true
+      secret: false
+    - default: false
+      description: The name of the account.
+      isArray: false
+      name: account_name
```

Comment:
why accountID, account_name?
let's have a convention camelcase/snakify to all args in all commands.
@Arsenikr what is your preference?

Reply:
I know that usually ID doesn't appear in lowercase but I can change it, @Arsenikr ?